packages/render-fidelity-tools/test/README.md

```diff
@@ -116,3 +116,48 @@ Quick Look skybox in renderers that we wish to appear at least somewhat
 comparable to Quick Look: [quick_1k.png](../../examples/assets/quick_1k.png).
 
 
+## Adding CLI renderer backends
+
+The `command` property of the renderer configuration allows adding external renderer processes to the screenshot generation and fidelity testing.
+
+```json
+"renderers": [
+    {
+      "name": "cli_renderer",
+      "description": "An offline rendering backend",
+      "command": { // providing an empty object will skip the renderer for image generation, but it is still expected to provide renderings in the golden folders for image comparison and report generation
```

Comment:
technically JSON is not allowed to contain comments. Might be better to move this up.